tests/phpunit/bootstrap.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
I meant using the `vendor/autoload.php` as the bootstrap and simply omitting the bootstrap.php until we actually need for something.